xarray/tests/test_computation.py

```diff
@@ -1004,6 +1004,24 @@ def arrays_w_tuples():
     return arrays, array_tuples
 
 
+@pytest.mark.parametrize("ddof", [0, 1])
+@pytest.mark.parametrize(
+    "da_a, da_b",
+    [arrays_w_tuples()[1][3], arrays_w_tuples()[1][4], arrays_w_tuples()[1][5]],
```

Comment:
I think this is missing the case where one dataset has NaNs but not the other. Is that right?

Reply:
Yep, will fix now!